src/mesh/parallel/fci.cxx

```diff
@@ -57,10 +58,13 @@ FCIMap::FCIMap(Mesh& mesh, int offset_, BoundaryRegionPar* boundary, bool zperio
     throw BoutException("FCIMap called with offset = 0; You probably didn't mean to do that");
   }
 
-  interp = InterpolationFactory::getInstance().create(&map_mesh);
+  auto interpolation_options = options["xzinterpolation"];
+  interp = XZInterpolationFactory::getInstance().create(&interpolation_options, &map_mesh,
+                                                        &interpolation_options);
```

Comment:
Passing the same variable twice is maybe a little surprising. I think I understand why, but I'll see if there's a cleaner way. Not the end of the world if there isn't though

Reply:
This issue has gone away with removing the `Options` argument to the `XZInterpolation` constructor.